sycl/test-e2e/ESIMD/unified_memory_api/Inputs/block_store.hpp

```diff
@@ -219,6 +218,164 @@ bool testACC(queue Q, uint32_t Groups, uint32_t Threads,
   return Passed;
 }
 
+template <typename T, uint16_t N, bool UseMask, bool CheckProperties,
+          typename StorePropertiesT>
+bool testSLM(queue Q, uint32_t Groups, StorePropertiesT StoreProperties) {
+  using shared_allocator = sycl::usm_allocator<T, sycl::usm::alloc::shared, 16>;
+  using shared_vector = std::vector<T, shared_allocator>;
+  constexpr uint16_t GroupSize = 8;
+
+  uint32_t Size = Groups * GroupSize * N;
+
+  std::cout << "SLM case: T=" << esimd_test::type_name<T>() << ",N=" << N
+            << ",UseMask=" << UseMask << ",UseProperties=" << CheckProperties
+            << std::endl;
+
+  sycl::range<1> GlobalRange{Groups};
+  sycl::range<1> LocalRange{GroupSize};
+  sycl::nd_range<1> Range{GlobalRange * LocalRange, LocalRange};
+
+  constexpr size_t Alignment = getAlignment<T, N, UseMask>(StoreProperties);
+
+  shared_vector Out(GroupSize, shared_allocator{Q});
+  T Out_val = esimd_test::getRandomValue<T>();
+  for (int i = 0; i < Size; i++)
+    Out[i] = Out_val;
```

Comment:
mix of styles in 1 var
```suggestion
  T OutVal = esimd_test::getRandomValue<T>();
  for (int i = 0; i < Size; i++)
    Out[i] = OutVal;
```